RangePicker/RangePicker.js

```diff
@@ -250,11 +264,12 @@ const RangePickerBase = kind({
 	},
 
 	styles: {
-		css,
-		className: 'rangePicker'
+		css: pickerTitleCss,
+		publicClassNames: ['inlineTitle', 'title']
```

Comment:
In the case of RangePicker, it may be better to combine classes in computed  instead of using publicClassNames.

Reply:
I changed the implementation to use mergeClassNameMaps  instead of publicClassNames